examples/app-basic/src/components/App/App.tsx

```diff
@@ -1,13 +1,27 @@
 import * as React from 'react';
 import { AppRootProps } from '@grafana/data';
+import { Route, Routes } from 'react-router-dom';
 import { PluginPropsContext } from '../../utils/utils.plugin';
-import { Routes } from '../Routes';
+import { PageOne } from '../../pages/PageOne';
+import { PageTwo } from '../../pages/PageTwo';
+import { PageThree } from '../../pages/PageThree';
+import { PageFour } from '../../pages/PageFour';
+import { ROUTES } from '../../constants';
 
 export class App extends React.PureComponent<AppRootProps> {
   render() {
     return (
       <PluginPropsContext.Provider value={this.props}>
-        <Routes />
+        <Routes>
```

Comment:
Rather than prefixing all routes' urls, can we provide a [basename](https://reactrouter.com/en/main/router-components/router) here instead?

Reply:
Hmm, as far as I understand the paths are relative by default in v6, so we don't need to prefix them or to set a basename - or did I misunderstand something?